### PR TITLE
chore: update golangci-lint

### PR DESCRIPTION
**Pull request by tam7t**
**What this PR does / why we need it**:

Updates golangci-lint and addresses findings introduced by the update

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ linters:
     - gofmt
     - goimports
     - gosimple
-    - golint
+    - revive
```
Comment (aramase) [inline .golangci.yml:14]: Could we move this down to place it in alphabetical order?

Comment (tam7t) [reply .golangci.yml]: sure!

**Review by aramase — COMMENTED** [review]
/lgtm

**Pull request merged**